src/guiguts/html_convert.py

```diff
@@ -685,6 +685,8 @@ def reset_ibs_dict() -> None:
                 maintext().replace(
                     f"{line_start}-2l lineend", f"{line_start}-1l lineend", "</h2>\n"
                 )
+                # Don't want footnote anchors in auto ToC
+                chap_heading = re.sub(r"\[.{1,5}\]", "", chap_heading)
```

Comment:
Should there be a warning in the manual that square brackets in the auto-ToC will be treated as footnote anchors, for the very rare cases where there might be legitimate square brackets in a title that is included in the autogenned ToC?